test/e2e/shared/exec_test.go

```diff
@@ -46,8 +47,8 @@ func Test_commandsForMachine(t *testing.T) {
 		{
 			name: "",
 			args: args{
-				machineIP: "10.6.0.230",
-				bastionIP: "172.24.4.58",
+				machineIP: "10.6.0.209",
+				bastionIP: "172.24.4.10",
```

Comment:
I am not sure why we need such change? do we have hard dependency on IP?

Reply:
This was just for debugging locally. The test is not executed (t.Skip() above)
It's just way easier to develop the exec method locally when you don't have to  execute the whole e2e tests to do it :)

I'll rollback the IPs, I just changed them when debugging the func

Reply:
I am just curious as, not a big deal

Reply:
Absolutely fine :)